js/interface.js

```diff
@@ -91,6 +86,7 @@ tinymce.init({
 
 // 1. Fired from Fliplet Studio when the external save button is clicked
 Fliplet.Widget.onSaveRequest(function() {
+  dataSourceProvider.forwardSaveRequest();
```

Comment:
shouldn't you remove the `$('form').submit();` here and only do it when the data source provider comes back with the result?

Reply:
I see you corrected this after I asked. Can you confirm if you have received the relevant induction from your colleagues? I need to make sure I address the reasons behind why you have made such "mistake".

Reply:
i already discussed with @YaroslavOvdii this comment and we solved all questions